Dispatchfile

```diff
@@ -0,0 +1,71 @@
+#!starlark
+# vi:syntax=python
+#
+
+git = "src-git"
+
+def secretVar(name, key):
+    return k8s.corev1.EnvVarSource(secretKeyRef=k8s.corev1.SecretKeySelector(localObjectReference=k8s.corev1.LocalObjectReference(name=name), key=key))
+
+gitResource(git,
+    url="$(context.git.url)",
+    revision="$(context.git.commit)")
+
+dindTask("publish", inputs=[git], steps=[
+  v1.Container(
+    name = "publish-charts",
+    workingDir =  "/workspace/src-git",
+    args = [
+      "/bin/bash", "-c",
+      """
+      apt-get update
```

Comment:
Is there a way to produce or rather use an image that already has the required packages installed? This would dramatically speed up running these tasks.

Reply:
@makkes Reasoning behind why I chose to do it this way:
Every time the K8S_VERSION variable gets updated, the contributor would need to also make sure to build a new image. I don't find this reliable as people are bound to forget!
That Dockerfile would also only be 2 lines. It's only taking the kindest/node image and adding a file to it. So between just pulling a pre-built image vs pulling the kindest/node image and adding a file to it, I don't think the difference in speed is significant.

Reply:
@cprovencher The difference in speed will be caused by not having to `apt-get update` and `apt get install [...]` every time we publish, test or lint the code (i.e. on every PR). Please consider having a Docker image for that.